15-eco.Rmd

```diff
@@ -25,44 +25,37 @@ library(vegan)             # community ecology package
 
 ## Introduction
 
-In this chapter we will model the floristic gradient of fog oases to reveal distinctive vegetation belts that are clearly controlled by water availability.
-To do so, we will bring together concepts presented in previous chapters and even extend them (Chapters \@ref(spatial-class) to \@ref(geometry-operations) and  Chapters \@ref(gis) and \@ref(spatial-cv)).
+This chapter models the floristic gradient of fog oases to reveal distinctive vegetation belts that are clearly controlled by water availability.
+The example, building on previous applied chapters, provides an opportunity to bring together and extend concepts presented in previous, to further enhance your skills at using R for geocomputation.
 
-Fog oases are fascinating vegetation formations, locally termed *lomas*, which develop on mountains along the coastal deserts of Peru and Chile.^[Similar vegetation formations develop also in other parts of the world, e.g., in Namibia and along the coasts of Yemen and Oman [@galletti_land_2016].]
-The deserts' extreme conditions and remoteness provide the habitat for a unique ecosystem, including species endemic to the fog oases.
-Despite the arid conditions and low levels of precipitation of around 30-50 mm per year on average, fog deposition increases the amount of water available to plants during austral winter.
-This results in green southern-facing mountain slopes along the coastal strip of Peru (Figure \@ref(fig:study-area-mongon)). 
+Fog oases, locally called *lomas*, are ecological systems found in the mountains along the coastal deserts of Peru and Chile.
```

Comment:
On the mountains? Because often these mountains are isolated and do not belong to a bigger mountain range.

Reply:
found on mountains along the...

Lomas can only develop on mountains where fog deposition can take place in greater magnitude.